index.js

```diff
@@ -88,9 +102,10 @@ datasets.forEach(dataset => {
         activityData: ['userID', 'date', 'flightsOfStairs', 'minutesActive', 'numSteps'],
         hydrationData: ['userID', 'date', 'numOunces'],
         bookings: ['userID', 'date', 'roomNumber'],
-        roomServices: ['userID', 'date', 'food', 'totalCost']
-      };
-
+        roomServices: ['userID', 'date', 'food', 'totalCost'],
+        destinations: ['id', 'destination', 'estimatedLodgingCostPerDay', 'estimatedFlightCostPerPerson', 'image', 'alt'],
+        trips: ['userID', 'destinationID', 'travelers', 'date', 'duration', 'status', 'suggestedActivities']};
```

Comment:
The POST will still go through if there's an extra parameter passed through the trips, even though they get a message returned saying they have an invalid parameter. Should the request fail if they have extra information passed in? @hannahhch